src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -343,6 +343,7 @@ final class RiskProvider: RiskProviding {
 			switch risk.level {
 			case .low:
 				store.shouldShowRiskStatusLoweredAlert = true
+				NotificationCenter.default.post(name: .riskStatusLowerd, object: nil)
```

Comment:
:/ Posting Notifications, is there really no other way to handle this?

Reply:
of course there is, but this was the most direct way for me, another way is to inject the risk provider in the exposure detection to detect this. but what is the problem with notifications?

Reply:
Multiple Reasons:

- Notifications are more like a shotgun solution, and should only be used if really everything in the App needs to be notified about something.
- It is very hard to see the Information flow, who did send out this Notification?
- Would be a new way in which we handle Informationflow inside the App, as you mentioned we most of the time use Injection or Combine. 
- One Notification might not seem harmful but image the the whole app using notifications, it just becomes a mess and very hard to debug (race conditions, timing issues, clarity of the Information flow, etc)
- In my personal opinion Notifications are a anti-pattern

Reply:
They make sense for 'global' stuff like event tracking. This could be a closure passed by the 'listener'.

Reply:
Replace notification with a risk publisher in the homeState of the ExposureDetectionViewModel.

Reply:
Need to refactor even this. Because the ExposureDetectionViewModel is not allocated all the time and we could miss a risk change.